static/app/views/performance/transactionSummary/transactionEvents/eventsTable.tsx

```diff
@@ -330,7 +345,11 @@ class EventsTable extends Component<Props, State> {
                   }}
                   location={location}
                 />
-                <Pagination pageLinks={pageLinks} />
+                <Pagination
+                  disabled={!paginationCaption}
```

Comment:
This condition is a little strange since it's already covered by links, is there are reason you made `disabled` based on caption?

Reply:
Ohh, you're definitely right. Just a brain fart, was thinking that disabled would disable the caption. PageLinks makes a bit more sense, if the query fails for the totalEvents, setting the caption to undefined should do the trick to disable the caption.